src/components/leaflet/LeafletMap.jsx

```diff
@@ -1,7 +1,20 @@
 import { MapContainer, TileLayer, Marker } from 'react-leaflet';
 import style from './leaflet.module.scss';
+import { Icon } from 'leaflet';
+import MarkerIcon from '../../assets/images/icon-location.svg';
+import ShadowIcon from '../../assets/images/marker-shadow.png';
```

Comment:
Is this the shadow for an old icon? How does it fit with a custom icon?

Reply:
removed